generators/app/templates/client/server.js

```diff
@@ -0,0 +1,22 @@
+require('./config/env');
+var path = require('path');
+var url = require('url');
+var proxy = require('proxy-middleware');
+var history = require('connect-history-api-fallback');
+var serveStatic = require('express').static;
+var server = require('express')();
+
+var port = process.env.NODE_ENV == 'production' ? process.env.PORT : 8080;
+
+server.use(history());
+server.use('/', serveStatic(path.join(__dirname, 'public')));
+
+if(process.env.NODE_ENV !== 'production') {
+  server.use('/', proxy(url.parse(process.env.WEBPACK_DEV_SERVER_URL)))
+}
+
+server.use('/', proxy(url.parse(process.env.API_URL)));
```

Comment:
But this is local proxying. It's what any serious server like Apache or nginx usually do. And probably what heroku or any other platform of the likes do behind the curtain.

Our other options would be to serve the SPA with Rails, instead of express, or really separate it into two different apps, which I think is totally overkill. We would have to have two apps on heroku for every app.

It makes sense on API's meant to be consumed by other clients, but not in our case, where the client of the app is the only client we intend to have.